src/test/scala/com/campudus/tableaux/testtools/RequestCreation.scala

```diff
@@ -5,26 +5,46 @@ import org.vertx.scala.core.json.Json
 
 object RequestCreation {
 
-  sealed abstract class ColType(val kind: String) {
+  object Columns {
+    def apply(): Columns = {
+      new Columns(Seq.empty)
+    }
+  }
+
+  case class Columns(columns: Seq[ColumnType]) {
+    def add(column: ColumnType): Columns = {
+      Columns(columns.:+(column))
+    }
+
+    def getJson: JsonObject = Json.obj("columns" -> columns.map(_.getJson))
+  }
+
+  sealed abstract class ColumnType(val kind: String) {
```

Comment:
I used `ColType` in the tests to avoid confusion with the already existing `ColumnType` of the implementation.